models/security-scheme.go

```diff
@@ -0,0 +1,93 @@
+package models
+
+import "encoding/json"
+
+// SecurityScheme maps AsyncAPI "SecurityScheme" object
+type SecurityScheme struct {
```

Comment:
This is another example of merging different schemas (those specified in `oneOf`) into a single one.